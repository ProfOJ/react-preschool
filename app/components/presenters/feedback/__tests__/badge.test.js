import React from 'react'
import Badge from '../badge.jsx'
import renderer from 'react-test-renderer'

it('renders a badge with the appropriate classes', () => {
  const component = renderer.create(<Badge animationDelay='0.5s' isFilled={true} shouldAnimate={true} />)

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
}
)
