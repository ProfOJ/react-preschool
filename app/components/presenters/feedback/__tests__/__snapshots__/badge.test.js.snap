// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a badge with the appropriate classes 1`] = `
<li
  className="star star--filled spin-in"
  style={
    Object {
      "animationDelay": "0.5s",
    }
  }
>
  <i>
    <svg
      vectorEffect="non-scaling-stroke"
      viewBox="0 0 51 48"
    >
      <use
        x="0"
        xlinkHref="#star"
        y="1"
      />
    </svg>
  </i>
</li>
`;
